scss/icon/_icons.helpers.scss

```diff
@@ -7,12 +7,15 @@
 	$size: $icon-size,
 	$style: $icon-style,
 	$position: "before",
-	$partial: false
+	$partial: false,
+	$defaultFontProperties: true
 ) {
 	&::#{$position} {
-		--icon-font-family: "#{"icons-" + $size + "-" + $style}",
-			"missing-icons";
-		--icon-font-size: #{to-rem($pxValue: $size)};
+		@if $defaultFontProperties {
+			--icon-font-family: "#{"icons-" + $size + "-" + $style}",
+				"missing-icons";
+			--icon-font-size: #{to-rem($pxValue: $size)};
+		}
```

Comment:
I like the idea making this optional, but in that case we could as well define it as a fallback to the CSS Variables reference on the following code lines:
https://github.com/db-ui/base/blob/cf16a3bd1e2078e5bf8209dbcc44884bc112c5b2/scss/icon/_icons.helpers.scss#L38L40